test/JLM_Tests/playlist.xml

```diff
@@ -1040,6 +1040,33 @@
 		</subsets>
 	</test>
 
+	<test>
+		<testCaseName>testOpenJ9DiagnosticsMXBean</testCaseName>
+		<variations>
+			<variation>Mode100</variation>
+			<variation>Mode101</variation>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
```

Comment:
Please run test in REPORTDIR as the test may generate core file. 

```
$(MKTREE) $(REPORTDIR); \
cd $(REPORTDIR); \
```

Reply:
@llxia I have made these changes, please review. Thanks